app/(site)/Om-oss/page.tsx

```diff
@@ -0,0 +1,21 @@
+import BackButton from '@/components/UI/backbutton';
+import Footer from '@/components/footer/footer';
+
+export default function LesmerPage() {
+    return (
```

Comment:
Should be an english page name => AboutUsPage

Reply:
Noted 📝